Theatre outfits

Davyei opened the pull request:
<!--
You are making this pull request for the Moffstation fork of Space Station 14.

Please be sure to follow general guidelines for upstream PRs, but also be sure to follow the Moffstation guidelines.
Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline
See the Harmony contributing guidelines for an example on what we want: https://github.com/ss14-harmony/ss14-harmony/blob/master/CONTRIBUTING.md
-->

## About the PR
<!-- What did you change? -->
- Adds new Sprite states for masks
- A new jester outfit resprite & accompanying green jester shoes
- Puts things in the loadout for clown, mime & musician!
- New Pierrot Improvements

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Honk Honk, drip!
also Loadouts needed to be sorted
the slippers should be making their funny gimmick noises.
## Technical details
<!-- Summary of code changes for easier review. -->

## Media
<!-- Attach media if the PR makes in-game changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="424" height="328" alt="Screenshot 2025-12-14 150907" src="https://github.com/user-attachments/assets/42dcaae8-cd95-4678-9d26-67fe7199cb9b" />
<img width="759" height="529" alt="Screenshot 2025-12-14 171712" src="https://github.com/user-attachments/assets/31a9a622-cd3c-41bc-a2e2-ee13446a4d23" />
<img width="434" height="575" alt="Screenshot 2025-12-15 183157" src="https://github.com/user-attachments/assets/b0414e65-2fa5-4320-b6e8-56a1e29a1ae8" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Upstream Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html) as well as the [Moffstation Contributing Guidelines](https://github.com/moff-station/moff-station-14/blob/master/CONTRIBUTING.md).
- [X] I have properly sectioned my changes into fork namespaces, and/or followed proper guidelines for modifying upstream files.
- [X] I have added media to this PR or it does not require an in-game showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl:
- add: new jester outfit & winter coat sprites
- tweak: pierrot outfit improvement
- tweak: vox mask sprites

Davyei committed:
Pierrot v.2!

red accenting instead of orange, new pierrot cone hat & pierrot fur coat!

Davyei committed:
good vox masks & vox hoods!!

vox clown masks are now appealing & look like masks

clown hoods now have vox equipped sprite so they don't look silly in their tiny hoods

Davyei committed:
Jester Shoe update!

110% more jingly, now has inhands sprites instead of using the chef shoes

Davyei committed:
Tons of things!

- Vox Clown masks made into face masks rather than full head masks
- Sad, Cool & Scared Mime masks altered to be similar to the normal masks- New Jester Outfit
- Clown Wintercoat replaced with Clown Bath Rode
- Mime Masks & Clown Masks in the loadout options
- Musician Loadout Outfits added to Loadout Options
- Lizard/Duck Slippers & Banana Clown shoes given back their noises & Banana Clown Shoes altered to be fun but not incredibly broken.
- Sec Mime & Clown Mask altered to provide no protection but can now be selected in the loadout as the cool mime/clown masks

Davyei committed:
final touches

final things before Centronias slams my hands with the reviewing cane

Centronias commented on Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml:
```diff
@@ -3,6 +3,7 @@
   startingInventory:
     ClothingHeadHatJester: 1
     ClothingUniformJumpsuitJester: 1
+    ClothingShoesJesterAlt: 2 # Moffstation Start - New Theatre Outfits
```
Remove the preexisting opening to this guard on line 10.

Centronias commented on Resources/Prototypes/Entities/Clothing/Head/hats.yml:
```diff
@@ -1002,16 +1002,26 @@
   id: ClothingHeadHatJester
   name: jester hat
-  description: A hat with bells, to add some merriness to your head.
+  description: A hat to let you jingle your balls for a crowd to see.
```
Davyei...
```suggestion
  description: A hat with bells, to add some merriness to your head.
```

Davyei replied:
all good, like I said I have still changed it to be to do with the bells & juggling

Centronias commented on Resources/Prototypes/Entities/Clothing/Head/hats.yml:
```diff
@@ -1002,16 +1002,26 @@
 
 - type: entity
-  parent: ClothingHeadHatJester
+  parent: ClothingHeadBase
```
```suggestion
  parent: ClothingHeadBase # Moffstation - New Clown Outfit
```

Centronias commented on Resources/Prototypes/Entities/Clothing/Masks/masks.yml:
```diff
@@ -247,20 +247,21 @@
   parent: [ClothingMaskClown, BaseSecurityContraband]
   id: ClothingMaskClownSecurity
-  name: security clown wig and mask
+  name: cool clown wig and mask # Moffstation - Clown Mask renamed to be Cool
```
Probably will need to remove the contra status.

Centronias commented on Resources/Prototypes/Entities/Clothing/Masks/masks.yml:
```diff
@@ -665,20 +666,20 @@
   parent: [ClothingMaskMime, BaseSecurityContraband]
   id: ClothingMaskMimeSecurity
-  name: security mime mask
+  name: cool mime mask # Moffstation - Mime Mask renamed to be Cool
```
Contra status again.

Centronias commented on Resources/Prototypes/Entities/Clothing/Shoes/misc.yml:
```diff
@@ -62,17 +63,18 @@
       path: /Audio/Items/Toys/weh.ogg # Copied from lizard plush sound
       params:
-        volume: -5
+        volume: -10
```
```suggestion
        volume: -5
```
imo leave this -- they can only be eaten like three times.

Centronias commented on Resources/Prototypes/Entities/Clothing/Shoes/misc.yml:
```diff
@@ -62,17 +63,18 @@
       params:
-        variation: 0.02
+        variation: 0.06
+        volume: -10
```
```suggestion
        volume: -10
        # Moffstation - End
```

Centronias commented on Resources/Prototypes/Entities/Clothing/Shoes/specific.yml:
```diff
@@ -58,18 +60,20 @@
+      collection: FootstepSlip
+      params:
+        volume: -10
+        variation: 0.10
```
```suggestion
# Moffstation - Begin - New Banana Shoes update!
  - type: FootstepModifier
    footstepSoundCollection:
      collection: FootstepSlip
      params:
        volume: -10
        variation: 0.10
# Moffstation - End
```

Centronias commented on Resources/Prototypes/Loadouts/loadout_groups.yml:
```diff
@@ -606,6 +606,27 @@
+  minLimit: 0
+  loadouts:
+  - RuffleNeck
+  # Moffstation - End - Clown Clothes
```
Entirely new prototypes (ie. things where you do a top level `- type: ...`) need to be put into a `_Moffstation` directory.

Centronias commented on Resources/Prototypes/Loadouts/loadout_groups.yml:
```diff
@@ -706,6 +759,17 @@
+  - BlackShoes
+  - LaceupShoes
+  - CowboyShoes
+# Moffstation - End - Mime Outfits
```
New, move.

Centronias commented on Resources/Prototypes/Loadouts/loadout_groups.yml:
```diff
@@ -739,6 +817,16 @@
+  - LaceupShoes
+  - PerformerShoes
+  - CowboyShoes
+# Moffstation - End - Musician Outfits
```
New, move.

Centronias reviewed (changes requested):
I am sorry to ask it since I know it'll be a pain, but can the modified sprites be changed to instead be new RSIs and the yaml updated to reference the new sprites?

Davyei committed:
Update Resources/Prototypes/Entities/Clothing/Shoes/misc.yml

Co-authored-by: Centronias <charlie.t.santos@gmail.com>
Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Update Resources/Prototypes/Loadouts/role_loadouts.yml

Co-authored-by: Centronias <charlie.t.santos@gmail.com>
Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
requessted changes

theatre neck rather than mime & clown neck
humour removed 1984 1984 1984, whats the point anymore
namespacing & moving things into _Moffstation/

Davyei committed:
Localisation stuff

Davyei committed:
yml fix

I forgot to remove this :p

Davyei committed:
big silly

accidentally made duplicates :)

Davyei committed:
.rsi stuff

should remove all the original .rsi things as requested

Davyei committed:
final things

I forgot I added the vox sprite for the hood too
